Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
 Section 2
-1: The House of Representatives shall be composed of Members chosen every second Year by the People of the several States, and the Electors in each State shall have the Qualifications requisite for Electors of the most numerous Branch of the State Legislature.
+1: The House of Representatives shall be composed of Members chosen every second Year by the Citizens of the several States, and the Electors in each State shall have the Qualifications requisite for Electors of the most numerous Branch of the State Legislature.
 
-2: No Person shall be a Representative who shall not have attained to the Age of twenty five Years, and been seven Years a Citizen of the United States, and who shall not, when elected, be an Inhabitant of that State in which he shall be chosen.
+2: No Person shall be a Representative who shall not have attained to the Age of thirty Years, and been ten Years a Citizen of the State in which he shall be chosen, and who shall not, when elected, be an Inhabitant of that State in which he shall be chosen, or if the State should choose that its Representatives be in service of a specific geographical area within the State, be an Inhabitant of that geographical area.
 
-3: Representatives and direct Taxes shall be apportioned among the several States which may be included within this Union, according to their respective Numbers, which shall be determined by adding to the whole Number of free Persons, including those bound to Service for a Term of Years, and excluding Indians not taxed, three fifths of all other Persons.2  The actual Enumeration shall be made within three Years after the first Meeting of the Congress of the United States, and within every subsequent Term of ten Years, in such Manner as they shall by Law direct. The Number of Representatives shall not exceed one for every thirty Thousand, but each State shall have at Least one Representative; and until such enumeration shall be made, the State of New Hampshire shall be entitled to chuse three, Massachusetts eight, Rhode-Island and Providence Plantations one, Connecticut five, New-York six, New Jersey four, Pennsylvania eight, Delaware one, Maryland six, Virginia ten, North Carolina five, South Carolina five, and Georgia three.
+3: Representatives shall be apportioned among the several States which may be included within this Union, according to their respective Citizen Populations. The actual Enumeration shall be made within three Years after the first Meeting of the Congress of these United States, and within every subsequent Term of five Years, in such Manner as they shall by Law direct. The Number of Representatives shall not exceed one for every thirty thousand Citizens, nor shall it be less than one for every one hundred thousand Citizens, excepting that each State shall have at least one Representative.
```

Comment:
Thanks to the advances in civil liberties from the 1860's, no three-fifths compromise is needed, and I'm very glad to purge its remnants.

The requirement that there be at least one Representative for each 100,000 Citizens would significantly expand the House of Representatives; this is necessary, because as it stands, the average Representative represents nearly 1 million people, which is far too many to establish any meaningful relationship with even a small fraction of their people. This makes elections much more vulnerable to marketing campaigns, funded by special interests, and much less likely to result in Representatives who resist corruption long-term.